ocs_ci/ocs/managedservice.py

```diff
@@ -64,3 +64,17 @@ def update_pull_secret():
         exec_cmd(
             f"oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson={secret_file.name}"
         )
+
+
+def get_consumer_names():
+    """
+    Get the names of all consumers
```

Comment:
You can add a note that this should be ran on provider.

Reply:
Done